android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/WalletUri.java

```diff
@@ -0,0 +1,17 @@
+package com.solana.mobilewalletadapter.walletlib.authorization;
+
+import androidx.annotation.IntRange;
+import androidx.annotation.Nullable;
+
+public class WalletUri {
```

Comment:
Is `public` the correct visibility for this type? Not 100% sure, can it be package-scoped?